starstream-interpreter/src/lib.rs

```diff
@@ -15,12 +16,24 @@ use starstream_types::{Spanned, Type};
 
 /// Execute a program.
 pub fn exec_program(program: &TypedProgram) {
-    eval_block(&program.statements, &Default::default());
+    if let Some(function) = program
+        .definitions
+        .iter()
+        .find_map(|definition| match definition {
+            TypedDefinition::Function(function) => Some(function),
+        })
+    {
+        eval_function(function);
+    }
+}
+
+fn eval_function(function: &TypedFunctionDef) {
+    let _ = eval_block(&function.body, &Default::default());
```

Comment:
Future interpreter work:
- Accept specific function and arguments to execute
- ControlFlow enum or similar instead of bool